manifester/manifester.py

```diff
@@ -12,13 +12,14 @@
 
 class Manifester:
     def __init__(self, manifest_category, allocation_name=None, **kwargs):
+        if isinstance(manifest_category, dict):
+            self.manifest_data = manifest_category
```

Comment:
For consistency's sake, you'll probably want to wrap the passed dict in a Box, or check whether its a Box instance.

What's read from `settings.manifest_category.get` will be a Box instance.